src/styles/card.css

```diff
@@ -0,0 +1,85 @@
+@import "./stars.css";
+
+.card {
+  width: 200px;
+  background: white;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  padding: 0 16px;
+  padding-top: 32px;
```

Comment:
These 2 lines can be consolidated, but the dimensions are a bit off, should be 32px 16px 16px based on the mockup